variants/github_actions_ci/workflows/ci.yml

```diff
@@ -0,0 +1,157 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+      - production
```

Comment:
@lukeify added
```
  pull_request:
    types: [opened, synchronize, ready_for_review]
```
to his client project PR. This was a TIL for me. Have we used this elsewhere? Do we know what subset of events we should target (if any)?

Docs: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request

Reply:
I don't think we've used those specific events anywhere, and I don't think we actually want them - the only difference between these and the defaults is we're swapping `reopened` for `ready_for_review`; while we might not need CI to (re)run on `reopened`, I don't think it makes sense to be running it again when you mark a PR ready for review given we're already running it whenever changes are pushed up.